common/src/main/java/com/scottlogic/deg/generator/utils/SetUtils.java

```diff
@@ -16,21 +16,23 @@
 
 package com.scottlogic.deg.generator.utils;
 
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.Set;
+import java.util.*;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
 public class SetUtils {
-    public static <T> Set<T> union(Set<T> a, Set<T> b) {
+
+    private SetUtils() {
+        throw new IllegalStateException("Do not instantiate utility class");
```

Comment:
Why is this needed?

Reply:
The private constructor is present to prevent an instantiation of the class from outside of the class. This is because the class only contains static methods, so there is no good reason to ever instantiate it.

The exception thrown is to prevent instantiation inside the class.